internal/wazeroir/compiler.go

```diff
@@ -114,8 +114,17 @@ type compiler struct {
 		depth int
 	}
 	pc     uint64
-	f      *wasm.FunctionInstance
 	result CompilationResult
+
+	// Function information
+	body       []byte
```

Comment:
body is the WebAssembly binary format (since we also are writing native code it might help keep head straight). Also, anything here correlated with wasm.Module would be best to call out. particularly below here are types ( complete type section I guess) and funcs (the whole index namespace or just module-defined?)

these will help folks who haven't memorized the spec, yet ;)

If you already documented it in the same package, you can say "// XX holds the same data documented by CompilationResult.XX", so that someone can click navigate to the why.